packages/server/src/service-manager.ts

```diff
@@ -47,17 +47,14 @@ export class ServiceWorkerManager implements IServiceWorkerManager {
     if ((installedVersion && installedVersion !== VERSION) || !installedVersion) {
       // eslint-disable-next-line no-console
       console.info('New version, unregistering existing service workers.');
-      navigator.serviceWorker
-        .getRegistrations()
-        .then((registrations) => {
-          for (const registration of registrations) {
-            registration.unregister();
-          }
-        })
-        .then(() => {
-          // eslint-disable-next-line no-console
-          console.info('All existing service workers have been unregistered.');
-        });
+      const registrations = await navigator.serviceWorker.getRegistrations();
+
+      for (const registration of registrations) {
+        await registration.unregister();
```

Comment:
Maybe there is a way to unregister them all at once and wait with `Promise.all`?

Reply:
That's better, I agree. Fixed in 987b0d52c31e36c005d964addb65815dae371e4b